athena-msk/pom.xml

```diff
@@ -0,0 +1,197 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>aws-athena-query-federation</artifactId>
+        <groupId>com.amazonaws</groupId>
+        <version>2022.46.1</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>athena-msk</artifactId>
+    <name>Athena MSK Connector</name>
+    <version>2022.46.1</version>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <slf4jVersion>1.7.30</slf4jVersion>
+        <log4j2Version>2.16.0</log4j2Version>
+        <aws.lambda-java-log4j2.version>1.5.0</aws.lambda-java-log4j2.version>
```

Comment:
Remove `slf4jVersion`

Remove `log4j2Version` 

Remove `aws.lambda-java-log4j2.version`

All of these have definitions in the root pom.xml already and can just be used.

Reply:
Hi @henrymai , one query, just LambdaRoleARN is enough right? Do we need to add PermissionBoundary ?

Reply:
Yes, I think the LambdaRoleARN should be enough for now. But we may get a request for PermissionBoundary in the future. We don't have to deal with it right now.